vllm_ascend/utils.py

```diff
@@ -590,11 +590,12 @@ def dense_optim_enable() -> bool:
     return envs_ascend.VLLM_ASCEND_ENABLE_DENSE_OPTIMIZE
 
 
-def enable_sp() -> bool:
-    from vllm.config import get_cached_compilation_config
-
+def enable_sp(vllm_config=None) -> bool:
+    from vllm.config import get_current_vllm_config
+    if vllm_config is None:
+        vllm_config = get_current_vllm_config()
```

Comment:
Let's only import when vllm_config is None
```suggestion
    if vllm_config is None:
        from vllm.config import get_current_vllm_config
        vllm_config = get_current_vllm_config()
```

Reply:
fixed